graylog2-web-interface/src/components/content-packs/ContentPackSelection.jsx

```diff
@@ -158,7 +158,11 @@ class ContentPackSelection extends React.Component {
   render() {
     const entitiesComponent = Object.keys(this.state.filteredEntities || {}).map((entityType) => {
       const group = this.state.filteredEntities[entityType];
-      const entities = group.map((entity) => {
+      const entities = group.sort((a, b) => {
```

Comment:
Please use [`javascript-natural-sort`](https://github.com/Bill4Time/javascript-natural-sort) for sorting, as it is more readable and also takes care of some corner cases. There's also a method in `SortUtils` if you want to ignore case while comparing.